components/xr/portal/Portal.tsx

```diff
@@ -2,7 +2,7 @@
 import { Entity } from 'aframe-react'
 import React from 'react'
 import { useRouter } from 'next/router'
-
+import isExternalUrl from 'utils/isExternalUrl'
```

Comment:
```suggestion
// @ts-ignore
import isExternalUrl from 'utils/isExternalUrl'
```
I'm getting a ts error 2307 without this